ovn-tester/ovn_utils.py

```diff
@@ -438,28 +438,53 @@ def set_inactivity_probe(self, value):
             ("inactivity_probe", value),
         ).execute()
 
-    def lr_add(self, name):
+    def lr_add(self, name, ext_ids: Optional[Dict] = None):
+        ext_ids = {} if ext_ids is None else ext_ids
+
         log.info(f'Creating lrouter {name}')
-        uuid = self.uuid_transaction(partial(self.idl.lr_add, name))
+        uuid = self.uuid_transaction(
+            partial(self.idl.lr_add, name, external_ids=ext_ids)
+        )
         return LRouter(name=name, uuid=uuid)
 
-    def lr_port_add(self, router, name, mac, dual_ip=None):
+    def lr_port_add(
+        self, router, name, mac, dual_ip=None, ext_ids: Optional[Dict] = None
+    ):
+        ext_ids = {} if ext_ids is None else ext_ids
```

Comment:
We're missing:
```
options = {} if options is None else options
```

which causes:
https://cirrus-ci.com/task/5461921055375360?logs=test_openstack#L447
```
2023-10-25 11:40:24,426 | ovsdbapp.backend.ovs_idl.vlog |ERROR| attempting to write bad value to column options (ovsdb error: expected string, got <class 'NoneType'>)
```

@mkalcok I can squash it in if you agree with this change.

Reply:
Yes, thank you for noticing this. I seem to remember fixing this one already, not sure how it resurfaced. I'll take care of it.

Reply:
> Yes, thank you for noticing this. I seem to remember fixing this one already, not sure how it resurfaced. I'll take care of it.

No wonder you thought that, because apparently I regressed this in "cms/openstack: Set MTU for VM and GW ports."! Sorry about that and thank you for fixing.

Reply:
Thanks, looks good now!